### PR TITLE
Rename 'insert.batch-size' configuration property into 'write.batch-size'

### PR DESCRIPTION
**Pull request by sergey-melnychuk**

### CODE REVIEW

**Review by hashhar — APPROVED** [review]
LGTM % documentation?

Does the documentation also need an update? Or this is not yet documented?

**Comment by findepi** [comment]
thanks!

> Rename 'insert.batch-size' session property into

session -> configuration

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java [hunk 1]
```diff
@@ -118,17 +118,17 @@ public JdbcMetadataConfig setDomainCompactionThreshold(int domainCompactionThres
...
-    public int getInsertBatchSize()
+    @Max(MAX_ALLOWED_WRITE_BATCH_SIZE)
+    public int getWriteBatchSize()
     {
-        return insertBatchSize;
+        return writeBatchSize;
     }
 
-    @Config("insert.batch-size")
+    @Config("write.batch-size")
```
Comment (findepi) [inline plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java:127]: You can add `@LegacyConfig("insert.batch-size")` if this was already *released*.

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java [hunk 2]
```diff
@@ -69,10 +69,10 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
                         jdbcMetadataConfig.isTopNPushdownEnabled(),
                         false))
                 .add(integerProperty(
-                        INSERT_BATCH_SIZE,
-                        "Insert batch size",
-                        jdbcMetadataConfig.getInsertBatchSize(),
-                        value -> validateInsertBatchSize(value, MAX_ALLOWED_INSERT_BATCH_SIZE),
+                        WRITE_BATCH_SIZE,
+                        "Write batch size",
```
Comment (findepi) [inline plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java:73]: Synchronize this with `@ConfigDescription`

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java [hunk 3]
```diff
@@ -110,7 +110,7 @@ public static int getDomainCompactionThreshold(ConnectorSession session)
 
     public static int getInsertBatchSize(ConnectorSession session)
     {
-        return session.getProperty(INSERT_BATCH_SIZE, Integer.class);
+        return session.getProperty(WRITE_BATCH_SIZE, Integer.class);
```
Comment (findepi) [inline plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java:113]: rename `getInsertBatchSize` too

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java [hunk 4]
```diff
@@ -69,10 +69,10 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
...
                         false))
                 .add(integerProperty(
-                        INSERT_BATCH_SIZE,
-                        "Insert batch size",
-                        jdbcMetadataConfig.getInsertBatchSize(),
-                        value -> validateInsertBatchSize(value, MAX_ALLOWED_INSERT_BATCH_SIZE),
+                        WRITE_BATCH_SIZE,
+                        "Write batch size",
+                        jdbcMetadataConfig.getWriteBatchSize(),
+                        value -> validateInsertBatchSize(value, MAX_ALLOWED_WRITE_BATCH_SIZE),
```
Comment (findepi) [inline plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java:75]: inline MAX_ALLOWED_WRITE_BATCH_SIZE in `validateInsertBatchSize`

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java (hunk 1 above)
Comment (sergey-melnychuk) [reply plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java:127]: https://github.com/trinodb/trino/pull/8434 is in Milestone 360, but I'm not sure if it was already released.
@hashhar knows better for sure :)

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java (hunk 2 above)
Comment (sergey-melnychuk) [reply plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java]: done

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java (hunk 4 above)
Comment (sergey-melnychuk) [reply plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java]: done

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java (hunk 3 above)
Comment (sergey-melnychuk) [reply plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java:113]: done

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java (hunk 1 above)
Comment (hashhar) [reply plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java:127]: It's not in any released version yet - 360 will be the first release to include it.

**Review by hashhar — APPROVED** [review]

**Pull request merged**